engines/registry/app/models/registry/authenticated_client.rb

```diff
@@ -20,7 +17,6 @@ def authenticate_by_system_credentials(login, password)
     @systems = System.get_by_credentials(login, password)
     if @systems.present?
       @account = login
-      @auth_strategy = :system_credentials
```

Comment:
good, now how do we know if `authenticate_by_system_credentials` succeeded or not ?